public/challenge-contributions/3/rpetrov.html

```diff
@@ -4,8 +4,211 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Challenge 3</title>
+
+  <style>
+    * {
+      border: 0;
+      padding: 0;
+      margin: 0;
+    }
+    html,
+    body {
+      min-height: 100vh;
+      background-color: #1d1d1d;
+      color: #fff;
+    }
+    .list {
+      display: grid;
+      grid-template-columns: repeat(auto-fit, minmax(min(200px, 100%), 1fr));
+      gap: 16px;
+      padding: 32px;
+      max-width: 1200px;
+      margin: auto;
+    }
+    .scene {
+      display: flex;
+      place-content: center;
+      perspective: 1000px;
+    }
+    .card {
+      --radii: 8px;
+      
+      box-sizing: border-box;
+      position: relative;
+      width: 100%;
+      aspect-ratio: 3/4;
+      border-width: 2px;
+      border-style: solid;
+      border-color: rgb(255, 255, 255, 0.4);
+      background-color: #000;
+      transition: transform 1s ease, border-color 0.5s ease, box-shadow 0.5s ease;
+      transform-style: preserve-3d;
+      border-radius: var(--radii);
+      box-shadow: 0 0 8px rgba(255, 255, 255, .4);
+    }
+    .card:hover {
+      transform: rotateY(180deg);
+      box-shadow: 0 0 8px rgba(152, 2, 182, 0.6);
+      border-color: rgba(152, 2, 182, 0.6);
+    }
+    .card img {
+      position: absolute;
+      top: 0;
+      left: 0;
+      display: block;
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+    .card__title {
+      font-family: system-ui, sans-serif;
+      font-size: 38px;
+      line-height: 1.2;
+      letter-spacing: 0.8px;
+      color: currentColor;
+    }
+    .card__face {
+      box-sizing: border-box;
+      position: absolute;
+      top: 0;
+      left: 0;
+      backface-visibility: hidden;
+      display: flex;
+      height: 100%;
+      width: 100%;
+      overflow: hidden;
```

Comment:
Why do you need to overflow: hidden here? Is backface-visibility: hidden; not enough?

Reply:
I need overflow hidden to cut out the rounded corners when I have an image inside, for example.
`backface-visibility: hidden;` is doing something different from `overflow: hidden;`, it removes the back of  an element which is transformed.